.gitignore

```diff
@@ -3,4 +3,6 @@ build/
 out/
 *.iml
 .idea
-.DS_Store
\ No newline at end of file
+.DS_Store
+ciinaboxes
```

Comment:
you need to not add excludes in this PR. I'm working on a change to rename the directory and add these...